backend/models/user.model.ts

```diff
@@ -28,6 +32,23 @@ const UserSchema: Schema = new Schema({
     required: true,
     enum: ["Volunteer", "Staff", "Admin"],
   },
+  startDate: {
+    type: Date,
+    required: true,
+  },
+  birthday: {
+    type: Date,
+    required: true,
+  },
+  locationType: {
+    type: String,
+    required: true,
+    enum: ["Remote", "Hybrid", "InPerson"],
+  },
+  headshot: {
+    type: String,
+    required: true,
+  },
```

Comment:
Same comment as before here: these probably shouldn't be `required` - the current signup functionality will break :(